packages/roosterjs-editor-plugins/lib/CustomReplace/CustomReplace.ts

```diff
@@ -91,7 +91,7 @@ export default class CustomReplacePlugin implements EditorPlugin {
         }
 
         // Exit early on input events that do not insert a replacement's final character.

```

Comment:
does this comment need to be updated? or still accurate?

Reply:
I feel it is still accurate.